### PR TITLE
[Meson] Add ndk-build to be part of meson @open sesame 12/07 09:04

### PR DESCRIPTION
**Pull request by zhoonit**
**Changes proposed in this PR:**
- Add option to build library using ndk (default disabled)
- Now related resources are downloaded in build directory

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -167,3 +167,30 @@ if get_option('enable-nnstreamer-tensor-filter')
...
+  jni_build_root = meson.build_root() / 'jni'
+
+  ndk_args = {
+    'NDK_PROJECT_PATH': jni_root,
+    'APP_BUILD_SCRIPT': jni_root / 'Android.mk',
+    'NDK_APPLICATION_MK': jni_root / 'Application.mk',
+    'NDK_LIBS_OUT': jni_build_root / 'libs',
+    'NDK_OUT': jni_build_root / 'objs',
+  }
+  ndk_additional_flags = ['-j2']
```
Comment (kparichay) [inline meson.build:183]: how about number of processors than just 2?

**Review by kparichay — APPROVED** [review]
LGTM other than the minor comment

File: meson.build (hunk 1 above)
Comment (zhoonit) [reply meson.build:183]: Do you have a number? Or I'll do 8.

Comment (kparichay) [reply meson.build:183]: I was thinking more like `grep -c '^processor' /proc/cpuinfo`

Comment (zhoonit) [reply meson.build]: Sound decent. Done

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**